### PR TITLE
Easier compare of visualfsharp

### PR DESCRIPTION
**Pull request by enricosada**
this merge some changes from visualfsharp repo

the whitespace cleanup is done to make easier to compare files of the visualfsharp repo 
now all differences with the `src` directory are bugfix, features or infrastructure

There is also the parallel pr Microsoft/visualfsharp#252 to visualfsharp repo

### CODE REVIEW

**Comment by enricosada** [comment]
wait to merge, i need to update it with two changes, see Microsoft/visualfsharp#252 comments

**Comment by enricosada** [comment]
ok, fixed as Microsoft/visualfsharp#252, ready to review and merge

File: src/fsharp/FSharp.Build/Microsoft.FSharp.Targets [hunk 1]
```diff
@@ -119,53 +118,53 @@ Copyright (C) Microsoft Corporation. Apache 2.0 License.
...
-      <_Temporary Remove="@(_Temporary)" />
+        <DocFileItem Include="$(DocumentationFile)" Condition="'$(DocumentationFile)'!=''"/>
     </ItemGroup>
 
-  </Target>
-
-  <ItemGroup>
-    <DocFileItem Include="$(DocumentationFile)" Condition="'$(DocumentationFile)'!=''"/>
-  </ItemGroup>
+    <ItemGroup Condition="'$(_DebugSymbolsProduced)' == 'true' and '$(PdbFile)' != ''">
```
Comment (dsyme) [inline src/fsharp/FSharp.Build/Microsoft.FSharp.Targets]: What's the change here?

**Comment by dsyme** [comment]
This looks good.  I've added one comment about what appears to be a non-whitespace change (sorry for the slow review, for some reason GitHub notifications for this repo weren't being sent to my email correctly)

**Pull request merged**